resources/js/features/downloads/components/DownloadableClientCard/DownloadableClientCard.tsx

```diff
@@ -52,7 +60,32 @@ export const DownloadableClientCard: FC<DownloadableClientCardProps> = ({ emulat
       className="flex h-full flex-col light:bg-white light:shadow-sm"
     >
       <BaseCardHeader>
-        <BaseCardTitle className="text-xl">{emulator.name}</BaseCardTitle>
+        <BaseCardTitle className="flex items-center gap-2 text-xl">
+          <span>{emulator.name}</span>
+
+          {!emulator.canDebugTriggers ? (
+            <BasePopover open={isPopoverOpen}>
+              <BasePopoverTrigger
+                onMouseEnter={() => setIsPopoverOpen(true)}
+                onMouseLeave={() => setIsPopoverOpen(false)}
+                onClick={(e) => e.preventDefault()}
+              >
+                <LuCircleAlert data-testid="warning-icon" className="text-neutral-500" />
+              </BasePopoverTrigger>
+
+              <BasePopoverContent
+                onMouseEnter={() => setIsPopoverOpen(true)}
+                onMouseLeave={() => setIsPopoverOpen(false)}
+                onPointerDownOutside={(e) => e.preventDefault()}
+                onInteractOutside={(e) => e.preventDefault()}
+              >
+                {t(
+                  'Developers may not be able to easily resolve tickets submitted from this emulator.',
```

Comment:
Tickets aren't submitted _from_ this emulator, maybe "for this emulator" or "against this emulator"?

The tickets aren't even about the emulator. Is there some way to say "for issues when using this emulator"?